### ISSUE TITLE
Windows Phone 8: Jumping container / rendering Issue 

### ISSUE THREAD

**Issue by tom-mayer** [jakiestfu/Snap.js#169]
I've been debugging this issue for about a day now, it is rather complicated so I will try to make it clear with some screenshots. First off, I really like the library, the fastest one I've seen out there! 

The issue only exists on Windows Phone8 IE10, not the desktop IE10. Your demo also has the same issue. I've tested it on a hardware WP8 and the latest WP8 Emulator Update 3. I am only using the open() function, not dragging or anything.

A call to open() starts the transition (transorm3d(266px,0,0)) is set on the covering element. 
easeCallback is called at the end of the transformation. 
Some screenshots to illustrate the problem:

The app in its normal state:
http://screencast.com/t/YnnMLz7qW

The button on the upper left is pressed, and the transition works. I pause the transition at the end of it. Note that the view that has scrolled to the left is at the right position.
http://screencast.com/t/mVFeEwa4sW

If I continue, the view jumps to the right, out of viewport for about 150px (I have tested this value by lowering the maxPosition property from 266 to under 100 so the jump occurs inside the viewport).
This is the part where it gets complicated to explain:
http://screencast.com/t/pezB0Xyf

The view has the button (to open/close) on the upper left, The view is rendered offscreen in the red area. So it is not visible. HOWEVER the keyword is _rendered_. The area marked with "A" is where the button for closing should have been rendered. If I click on that area the sidebar closes. 
As a matter of face this happens with all clickable elements. Their clickable area is positioned at the right point, but they are rendered about 150px off after the jump.

I hope this was somehow clear. I'm happy to conduct further tests or give further information if needed. Or help any other way ;).

**Comment by OliverKK** [comment]
This issue is reproducable by open the demo http://jakiestfu.github.io/Snap.js/demo/apps/default.html with any Windows Phone 8.0.

Same problem here.

@jakiestfu Are u still maintaining this project?

**Comment by saramgsilva** [comment]
i have the same issue.... someone solved it?

**Comment by teltploek** [comment]
I've submitted a pull-request, that I believe should fix this issue. Feel free to try it out for yourselves:

https://github.com/jakiestfu/Snap.js/pull/221

**Comment by lethek** [comment]
I'm also having the same issue on Windows Phone 8 (lumia 920), and also a very similar issue on my Android phone.

@teltploek I tried your pull-request but unfortunately it had no effect for me.

**Comment by teltploek** [comment]
You're right. I'm trying something else now. Will submit a new PR if I'll be able to fix it.

Old PR closed.
